go.mod

```diff
@@ -20,10 +20,10 @@ require (
 	k8s.io/apimachinery v0.17.0
 	k8s.io/cli-runtime v0.17.0
 	k8s.io/client-go v0.17.0
-	knative.dev/eventing v0.13.6
-	knative.dev/pkg v0.0.0-20200409225505-9147309d5366
-	knative.dev/serving v0.13.2
-	knative.dev/test-infra v0.0.0-20200409221904-5e04d955cdb9
+	knative.dev/eventing v0.13.7
```

Comment:
there seems to be a merge conflict ? 

close this? since it is generally generated by a bot /cc @rhuss  ?